project 1/styles.css

```diff
@@ -0,0 +1,70 @@
+body {
+    background-color: #DCE6D5;
+}
+
+
+h1 {
```

Comment:
Look for places where your styles are shared and can combine 'em:
```css
h1, h2 {
    color: #223809;
    font-family: 'Libre Baskerville', serif;
    letter-spacing: .3px;
    text-align: center;
}
```
(If you add the `font-family` to the `body`, you can skip that line here.)